### PR TITLE
Update Angular version & switch to SDK-browser

### PR DESCRIPTION
**Pull request by emilianofant-split**
# angular-sdk-examples

## What did you accomplish?
In this PR I'm introducing 2 big changes:

1 - Upgrade Angular version in the example, from 9 to 11.2.5
2 - Removed the old SDK client in favour of  the new SDK-browser version.

... and other small changes:
- Added `.nvmrc` file
- Upgraded other packages/dependencies
- Some fixes/additions in the documentation

Also, in the example we are using `InLocalStorage` capability. You should be able to see it working when correctly setting the example with a real APIKEY and key.

Screen Shot 2021-03-17 at 17 22 55

## How do we test the changes introduced in this PR?
Following the same instructions as the README states, to build and run the local app.

_Expected behaviour:_ You should see the app running and the Features being requested.

## Related tickets:
[Update Angular Example](https://splitio.atlassian.net/browse/SDKS-2479)
[Update Angular Example with new SDK Browser](https://splitio.atlassian.net/browse/SDKS-2480)

## Extra Notes
Fixing [this issue brought by a customer](https://github.com/splitio/angular-sdk-examples/pull/9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.1.
```
Comment (EmilianoSanchez) [inline README.md]: Emi, check if we have to update version here (9.0.1 -> 11.?)

File: README.md [hunk 2]
```diff
@@ -14,4 +14,4 @@ The goal of this project is to show one way to to set up split in Angular.
```
Comment (EmilianoSanchez) [inline README.md:19]: we could add some comment like the one [here](https://github.com/splitio/react-sdk-examples#running-the-example), to explain that users must update the Api key and check that evaluated splits exist in their workspace.

**Review by EmilianoSanchez — COMMENTED** [review]
Some comments to check in the README

File: README.md (hunk 1 above)
Comment (emilianofant-split) [reply README.md]: The thing is, the original codebase was generated using that version (9.0.1), but in order to serve the app with the new Angular version, you will have to update to 11. I'm not sure what to do here 😛

**Review by EmilianoSanchez — APPROVED** [review]

**Pull request merged**